### PR TITLE
BETA: Register niksne.is-a.dev

### PR DESCRIPTION
**Pull request by NikSneMC**
Added `niksne.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/niksne.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "NikSne",
+        "email": "niksne.mc@yandex.ru"
+    },
+    "record": {
+        "CNAME": "www.niksne.ru"
```
[the remark this answers is not in the document]
Comment (NikSneMC) [reply domains/niksne.json:7]: > I cannot reach your website
> 
> > [www.niksne.ru](http://www.niksne.ru) redirected you too many times.

Yup, my website is WIP now

**Comment by NikSneMC** [comment]
The site is reacheble now

**Comment by VaibhavSys** [comment]
> The site is reacheble now

You should reopen the PR

**Comment by NikSneMC** [comment]
> > The site is reacheble now
> 
> You should reopen the PR

#13245

**Pull request closed without merging**